Userland/Libraries/LibJS/Runtime/ArrayBuffer.cpp

```diff
@@ -57,6 +57,23 @@ bool ArrayBuffer::is_resizable_array_buffer() const
     return m_max_byte_length.has_value();
 }
 
+// 1.1.7 HostResizeArrayBuffer ( buffer, newByteLength ), https://tc39.es/proposal-resizablearraybuffer/#sec-hostresizearraybuffer
+ThrowCompletionOr<bool> ArrayBuffer::host_resize_array_buffer(GlobalObject& global_object, size_t new_byte_length)
```

Comment:
Since the spec uses `handled` and `unhandled` for this, can we return an enum with those member instead?